demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -423,6 +423,15 @@ interface TranscriptSegment {
   endTimeMs: number;
 }
 
+interface TranscriptEntityModel {
+  contentIdentificationType?: string;
+  contentRedactionType?: string;
+  enablePartialResultsStability?: boolean;
+  partialStabilityFactor?: string;
+  entityType?: string;
+  languageModel?: string;
```

Comment:
Are `enablePartialResultsStability`, `partialStabilityFactor`, `languageModel` related to TranscriptEntity`? Not sure whether we need to define the interface here.

Reply:
Actually they are not. Should be named something like "TranscriptionStreamParams". It is more a collection of parameters that are provided to start transcription based on the user selection. The main idea for this was to reduce the actual number of parameters being passed to the method startLiveTranscription(https://github.com/aws/amazon-chime-sdk-js/blob/0a6f73b60c2e24e5b46cf42dfc3ffa415efa6719/demos/browser/app/meetingV2/meetingV2.ts#L1092-L1096) . Let me know if "TranscriptionStreamParams" looks ok to you.

Reply:
Got it. Yeah that sounds more accurate. In that case do we need https://github.com/aws/amazon-chime-sdk-js/pull/1634/files#diff-e870111fa9bc964600df41f9dedbd7396bef0bd816432b2333d0e33da18c0b15R1031-R1035